app/views/signup/template.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
This is purely for adding the html format into the .erb files as they aren't recognised by vscode as html files.